Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -4358,15 +4370,15 @@ ObservableList<ControlProperty> IWindowExplorer.GetElementProperties(ElementInfo
             ObservableList<ControlProperty> list = new ObservableList<ControlProperty>();
 
             //Base properties 
```

Comment:
this not supposed to be POMElementProperty but regular ControlPropery and might be called from WindowsExplorer

Reply:
done